.zenodo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "DVC: Data Version Control - Git for Data & Models",
+  "keywords": [
+    "data-science", "data-version-control", "machine-learning", "git",
+    "developer-tools", "reproducibility", "collaboration", "ai", "python"],
+  "contributors": [
+    {"name": "DVC team", "type": "Other", "affiliation": "Iterative"}]
```

Comment:
Not sure if we want `name: "DVC team"` or `name: "Iterative"` or `name: "iterative.ai"` or something else... Same with `"affilitation"`.

Reply:
are there example from Google or other companies? what usually people do? :)

Reply:
fyi `affiliation` can be blank and will show up when `name` is hovered over.

`name` should really match whatever we choose for the citation text (i.e. think of it as an "author" field)